### DISCUSSION TITLE
PolarFire SoC Training Materials

### DISCUSSION THREAD

**Discussion by hughbreslin** [polarfire-soc/polarfire-soc-documentation#225] [Announcements]
We may already have training material that covers the question you are going to ask, please take the time to review the following resources before asking a question:

- [Training videos](https://www.youtube.com/c/MicrochipDeveloperHelp/playlists?view=50&sort=dd&shelf_id=4)
- [Webinar series](https://www.microchip.com/en-us/education/technical-learning-center/webinars/fpga-webinars/risc-v-innovation-unleashed)
- [Documentation](https://github.com/polarfire-soc/polarfire-soc-documentation)

**Comment by julian202** [comment]
The link to Training#1: Introduction to the PolarFire is broken.
